src/Audio_Registers.md

```diff
@@ -295,20 +305,27 @@ Period changes (written to `NR33` or `NR34`) only take effect after the followin
   "NR34" 7:"Trigger" 6:"Length enable" 2-0:"Period"
 }}
 
-- **Trigger** (*Write-only*): Writing any value to `NR34` with this bit set [triggers](<#Triggering>) the channel.
+- **Trigger** (*Write-only*): Writing any value to `NR34` with this bit set [triggers](<#Triggering>) the channel, causing the
+  following to occur:
+  * Ch3 is enabled.
+  * If length timer expired it is reset.
+  * The period divider is set to the contents of `NR33` and `NR34`.
+  * Volume is set to contents of `NR32` initial volume.
+  * Wave RAM index is reset, but its *not* refilled.
 
-  :::warning RETRIGGERING CAUTION
+:::warning RETRIGGERING CAUTION
 
-  On monochrome consoles only, retriggering CH3 while it's about to read a byte from wave RAM causes wave RAM to be corrupted in a generally unpredictable manner.
+On monochrome consoles only, retriggering CH3 while it's about to read a byte from wave RAM causes wave RAM to be corrupted in a generally unpredictable manner.
 
-  :::
+:::
 
-  :::warning PLAYBACK DELAY
+:::warning PLAYBACK DELAY
 
-  Triggering the wave channel does not immediately start playing wave RAM; instead, the *last* sample ever read (which is reset to 0 when the APU is off) is output until the channel next reads a sample.
-  ([Source](https://github.com/LIJI32/SameSuite/blob/master/apu/channel_3/channel_3_delay.asm))
+Triggering the wave channel does not immediately start playing wave RAM; instead, the *last* sample ever read (which is reset to 0 when the APU is off) is output until the channel next reads a sample.
+([Source](https://github.com/LIJI32/SameSuite/blob/master/apu/channel_3/channel_3_delay.asm))
+
+:::
 
-  :::
```

Comment:
Wouldn't unindenting those break the list in two?

Reply:
With the last commit the warnings are nested 2 levels deep in the sublist.